web/solutii/X_O/Mihai_Adascalitei/XO1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Nu trebuie sa adaugi un fisier in acest director. Trebuie respectata structura prezentat la laborator.
Ai fost pe aproape, directorul trebuie sa fie de forma `web/solutii/nume_prenume/*` tu ai pus in root-ul `web/solutii`.
Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).